integration-tests/test_man_page.py

```diff
@@ -32,19 +32,36 @@ def test_man_page_commands(command):
 
 
 @pytest.mark.parametrize(
-    "option",
+    "options",
     [
-        "--activation-key, -a",
-        "--organization, -o",
-        "--password, -p",
-        "--server",
-        "--username, -u",
+        ["--activation-key", "-a"],
+        ["--organization", "-o"],
+        ["--password", "-p"],
+        ["--username", "-u"],
+        ["--enable-feature", "-e"],
+        ["--disable-feature", "-d"],
+        ["--content-template", "-c"],
     ],
 )
-def test_man_page_connect_options(option):
-    """Test verifies if man page displays existing options for commands"""
+def test_man_page_connect_options(options):
+    """
+    Test verifies if man page displays existing options for commands
+    """
     command_op = subprocess.check_output(["man", "rhc"]).decode("utf-8")
-    if option == "--server":
-        assert option not in command_op
-    else:
+    for option in options:
         assert option in command_op
+
+@pytest.mark.parametrize(
+    "feature_id",
+    [
+        "content",
+        "analytics",
+        "remote-management",
+    ]
+)
+def test_man_page_feature_ids(feature_id):
+    """
+    Test verifies if man page contains IDs of features
+    """
+    command_op = subprocess.check_output(["man", "rhc"]).decode("utf-8")
```

Comment:
@jvlcek as you can see here: we run `man rhc` ... it means we try to parse content of man page installed to the system. We do not parse content of source code or man page generated by meson. We test content of rpm.